scc/building-from-source.md

```diff
@@ -449,3 +453,75 @@ Instead of a `GitRepository` object, an `ImageRepository` is created:
 `ImageRepository` provides the same semantics as `GitRepository`,
 except that it looks for source code in container image registries rather than
 Git repositories.
+
+## <a id="maven-artifact"></a>Maven Artifact
+
+This approach is intended to be used to aid integration with existing CI/CI
+systems (e.g.: Jenkins) and can pull artifacts from existing Maven repositories,
+including Jfrog Artifactory.
+
+At this time, there are no dedicated fields in the `Workload` resource for
+specifying the Maven artifact configuration; you need to fill in the
+`name`/`value` pairs in the `params` structure.
+
+Example:
+
+```yaml
+apiVersion: carto.run/v1alpha1
+kind: Workload
+metadata:
+  name: my-workload
+  labels:
+    app.tanzu.vmware.com/workload-type: web
+spec:
+  params:
+  - name: maven
+    value:
+    - name: groupId
+      value: com.example
+    - name: artifactId
+      value: spring-boot-initial
+    - name: version
+      value: RELEASE   # latest 'RELEASE' or a specific version (e.g.: '1.2.2')
+    - name: type
+      value: jar
+    - name: classifier # optional
+      value: sources   # optional
+```
+
+The `tanzu` CLI does not yet support creating workloads for Maven artifacts.
```

Comment:
Rob maybe we can add this bit: https://gitlab.eng.vmware.com/supply-chain-choreographer/catalog/-/merge_requests/94#context , (using the --param-yaml flag) for creating workloads with maven params